lib/build/combo.css

```diff
@@ -63,3 +63,9 @@ button.combo-button:focus {
 ul.combo-list li.disabled {
```

Comment:
Overordnet set tror jeg vi skal have en snak om hvordan grænsefladen fungerer for brugeren.
Hvad får man tilbage i callback'et hvis en rawvalue bliver valgt og hvordan matcher det de typer der er defineret i Combojs?